app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :find_post, only: %i[index create]
+
+  def index
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: @comment.errors, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
![image](https://github.com/meryemsanem/blog-app/assets/93039359/40273ab2-41d5-46f5-8d8b-85f544f8270b)

- [ ] Great work so far but when i tried to create a comment from the API then i got this error which you can see in the image above 😨 Kindly fix this error to make your app work as expected 😊